src/test/java/com/icthh/xm/ms/entity/util/IsCollectionNotContaining.java

```diff
@@ -33,7 +28,7 @@ public void describeTo(Description description) {
         description.appendText("a collection not containing ").appendDescriptionOf(this.elementMatcher);
     }
 
-    @Factory
+//    @Factory
```

Comment:
@osiievska23 check and remove it if not needed

Reply:
done